bears/java/JavaPMDBear.py

```diff
@@ -38,6 +38,7 @@ def check_prerequisites(cls):  # pragma: no cover
                         allow_unused_code=('check_unused', negate))
     def create_arguments(filename, file, config_file,
                          check_best_practices: bool = True,
+                         pmd_config: str = None,
```

Comment:
To structure the param list more or less, I would put this one line above :+1:

Reply:
Above check_best_practices?